src/bilder/images/edxapp/files/edx_web_playbook.yml

```diff
@@ -6,7 +6,16 @@
   vars:
     serial_count: 1
     CLUSTER_NAME: edxapp
-    NGINX_ENABLE_SSL: false
+    EDXAPP_LMS_PREVIEW_NGINX_PORT: 80
```

Comment:
Should this `yml` file be called `edxapp_web_playbook.yml`?

Reply:
Makes sense. I'll change it.